pantheon/src/main/java/tech/pegasys/pantheon/controller/CliquePantheonController.java

```diff
@@ -156,7 +156,11 @@ private CliquePantheonController(
             syncConfig.downloaderParallelism(),
             syncConfig.transactionsParallelism(),
             syncConfig.computationParallelism(),
-            metricsSystem);
+            metricsSystem,
+            syncConfig.getEthereumWireProtocolConfiguration().getMaxGetBlockHeaders(),
```

Comment:
Hey @abdelhamidbakhta - Just seeing this.  Can we update 2 things here:
-  Remove `EthProtocolConfiguration` from `SyncConfiguration`
- Just pass a single `EthProtocolConfiguration` to the protocol manager rather than extracting out each option and passing it individually to the constructor

Reply:
see PR
https://github.com/PegaSysEng/pantheon/pull/1255